docs/src/chains/osmosis.md

```diff
@@ -6,7 +6,7 @@ Osmosis is a cutting-edge decentralized exchange built on the Cosmos network, de
 
 ## Usage
 
-See how to setup your main function in the [main function](../single_contract/scripting.md#main-function) section. Update the network passed into the `Daemon` builder to be `networks::OSMOSIS_1`.
+See how to setup your main function in the [main function](../single_contract/scripting.md#main-function) section. Update the network passed into the `Daemon` builder to be `networks::OSMO_5`.
```

Comment:
:eyes:

Reply:
We don't have OSMOSIS_1 network yet - I think issue #168 requests it.